test/e2e/network_test.go

```diff
@@ -233,6 +231,11 @@ func testNorthSouthNetworking(t *testing.T) {
 	testCtx, err := NewTestContext()
 	require.NoError(t, err)
 
+	// Ignore the application ingress load balancer test for None and vSphere platforms as it has to be created manually
```

Comment:
Can you pls add back the below comment from the delete part? Not sure if this is going to be useful but  don't want to miss out the link if it is useful.